src/main/resources/liquibase/changelog/20191206123004_changelog.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="user (generated)" id="1575631812464-1">
+        <addColumn tableName="field_config">
+            <column name="pattern" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="user (generated)" id="1575623647611-2">
+        <update tableName="field_config">
+            <column name="format" value="$1 $2 $3"/>
+            <where>name='phoneNumber'</where>
+        </update>
+        <update tableName="field_config">
+            <column name="format" value="$1 $2 $3"/>
+            <where>name='alternatePhoneNumber'</where>
+        </update>
+        <update tableName="field_config">
+            <column name="format" value="$1 $2 $3"/>
+            <where>name='phone'</where>
+        </update>
+        <update tableName="field_config">
+            <column name="pattern" value="(\d{0,3})(\d{0,3})(\d{4})"/>
```

Comment:
I think something like this should work:`^(\\+?\\d{2,3})??\\s*(\\d{3,5})\\s*(\\d{4})$`, but I'm not sure how long the groups should be, also you need to remember to use double `\\` when creating regex from string.